app/serializers/api/v1/contact_serializer.rb

```diff
@@ -1,7 +1,15 @@
 module Api::V1
   class ContactSerializer < ApplicationSerializer
     embed :ids, include: true
-    attributes :id, :name, :mobile
+    attributes :id, :name, :mobile, :mobile_phone_number
     has_one :address, serializer: AddressSerializer
+
+    def mobile_phone_number
```

Comment:
@abulsayyad123  Why do we need to send ** mobile_phone_number** when we're already sending ** mobile??**